src/url.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import parse from './parse';
+import format from './format';
+import { resolve, resolveObject } from './resolve';
+
+class Url {
+  static parse = parse;
+  static format = format;
+  static resolve = resolve;
+  static resolveObject = resolve;
+}
+
+export default Url;
```

Comment:
Might be worth skipping `class` here, since the output is [huge](https://babeljs.io/repl#?browsers=IE%3E%3D11&build=&builtIns=usage&spec=false&loose=false&code_lz=MYGwhgzhAECqBOJoG8BQ1oQC5iwS2GgAcx4IBTaAXmNIoG51Md9CAzAe3gFtdrpOPXIwzZcBaPHIQOIAG6UaUmfPIjm4wstkKA8gCMAVuWBZ-21YwC-QA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=env%2Cstage-0%2Cenv&prettier=true&targets=&version=7.9.0&externalPlugins=%40babel%2Fplugin-transform-runtime%407.9.0). Could do this instead:

```suggestion
export default function Url () {}
Url.parse = parse;
Url.format = format;
Url.resolve = resolve;
Url.resolveObject = resolve;
```